ui/package.json

```
@@ -16,7 +16,7 @@
 		"marked": "^7.0.2",
 		"monaco-editor": "^0.41.0",
 		"plotly.js-dist-min": "^2.22.0",
-		"remixicon": "latest",
+		"remixicon": "4.1.0",
```

Can we keep on latest? They keep adding icons and then people want to use them, but can't find them

---

I just don't think `latest` keyword is valid. And when you install dependencies this way it will install version `3.5.0` which definitely is not `latest`. Current version of this library is 4.1.0. If we want to have always the newest version then we should use
```suggestion
		"remixicon": "*",
```